src/Data/Aeson/Extra/Merge.hs

```diff
@@ -51,3 +39,21 @@ mergeA :: Functor f
       => (forall a. (a -> a -> f a) -> ValueF a -> ValueF a -> f (ValueF a))
       -> Value -> Value -> f Value
 mergeA f a b = embed <$> f (mergeA f) (project a) (project b)
+
+-- | Example of using 'merge'. see <https://lodash.com/docs#merge>:
+--
+-- /Note:/ not tested against JavaScript lodash, so may disagree in the results.
+--
+-- @since 0.4.1.0
+lodashMerge :: Value -> Value -> Value
+lodashMerge = merge alg
```

Comment:
Lodash is most popular library on https://www.npmjs.com/, and the docs link was the same from the moment `Merge` module was introduced here (1.5 years ago). 

Please, don't generalize. Bashing JS won't help to change my mind.